contact-form/assets/index.html

```diff
@@ -0,0 +1,168 @@
+            ><span>Contact form with SendGrid</span>
+            <aside>
+              <svg
+                class="icon"
+                role="img"
+                aria-hidden="true"
+                width="100%"
+                height="100%"
+                viewBox="0 0 20 20"
+                aria-labelledby="NewIcon-1577"
+              >
+                <path
+                  fill="currentColor"
+                  fill-rule="evenodd"
+                  d="M6.991 7.507c.003-.679 1.021-.675 1.019.004-.012 2.956 1.388 4.41 4.492 4.48.673.016.66 1.021-.013 1.019-2.898-.011-4.327 1.446-4.48 4.506-.033.658-1.01.639-1.018-.02-.03-3.027-1.382-4.49-4.481-4.486-.675 0-.682-1.009-.008-1.019 3.02-.042 4.478-1.452 4.49-4.484zm.505 2.757l-.115.242c-.459.9-1.166 1.558-2.115 1.976l.176.08c.973.465 1.664 1.211 2.083 2.22l.02.05.088-.192c.464-.973 1.173-1.685 2.123-2.124l.039-.018-.118-.05c-.963-.435-1.667-1.117-2.113-2.034l-.068-.15zm10.357-8.12c.174.17.194.434.058.625l-.058.068-1.954 1.905 1.954 1.908a.482.482 0 010 .694.512.512 0 01-.641.056l-.07-.056-1.954-1.908-1.954 1.908a.511.511 0 01-.71 0 .482.482 0 01-.058-.626l.058-.068 1.954-1.908-1.954-1.905a.482.482 0 010-.693.512.512 0 01.64-.057l.07.057 1.954 1.905 1.954-1.905a.511.511 0 01.71 0z"
+                ></path>
+              </svg>
+              Live
+            </aside>
+          </a>
+        </nav>
+      </header>
+    </div>
+    <main>
+      <div class="content">
+        <section>
+          <h2>Contact form</h2>
+          <p>
+            This is an example of a contact form you can use on your own site.
+            Rather than expose your email address, it uses a
+            <a href="https://www.twilio.com/docs/runtime/functions"
+              >Twilio Function</a
+            >
+            and the
+            <a href="https://www.twilio.com/sendgrid/email-api">SendGrid API</a>
+            to send the email.
+          </p>
+          <hr />
+          <form action="/send-email" method="POST" id="contact-form">
+            <div class="form-field">
+              <label for="email">Your email address</label>
+              <input
+                type="email"
+                inputmode="email"
+                name="from"
+                id="from"
+                required
+              />
+            </div>
+            <div class="form-field">
+              <label for="subject">Subject</label>
+              <input type="text" name="subject" id="subject" required />
+            </div>
+            <div class="form-field">
+              <label for="content">Your message</label>
+              <textarea name="content" id="content" required></textarea>
+            </div>
+            <div class="actions">
+              <button type="submit" id="submit-button" class="button-primary">
+                Send
+              </button>
+            </div>
+          </form>
+          <p id="status" class="status" hidden></p>
+        </section>
```

Comment:
You could add the `<!-- EDIT_CODE_V2 -->` snippet here.

Reply:
Ah! I knew there was something I'd forgotten!